salt/metalk8s/addons/prometheus-operator/deployed/chart.sls

```diff
@@ -59893,7 +59896,7 @@ spec:
       expr: (kube_node_spec_taint{job="kube-state-metrics",key="node.kubernetes.io/unreachable",effect="NoSchedule"}
         unless ignoring(key,value) kube_node_spec_taint{job="kube-state-metrics",key=~"ToBeDeletedByClusterAutoscaler|cloud.google.com/impending-node-termination|aws-node-termination-handler/spot-itn"})
         == 1
-      for: 5m
+      for: 15m
```

Comment:
Note: I can't figure out where this comes from - it doesn't look like that's from my diff, perhaps a value change that occurred without regeneration?

Reply:
yes it is part of your commit, perhaps a small typo ??
https://github.com/scality/metalk8s/commit/88030820f89ecbdeeefce932b0ad12142c479d54

Reply:
Looks like re-rendering the chart requires some 'special care': see the commit message of https://github.com/scality/metalk8s/commit/1c88876d2100baf04c898232f30531ab62f8f3c8